modules/nw-multus-ipam-object.adoc

```diff
@@ -276,13 +276,13 @@ ipamConfig:
 <1> A collection of mappings that define IP addresses to assign to the virtual
 interface. Both IPv4 and IPv6 IP addresses are supported.
 
-<2> An IP address that you specify.
+<2> An IP address and network prefix that you specify. For example, if you specify `10.10.21.10/24`, then the additional network is assigned an IP address of `10.10.21.10` and the netmask is `255.255.255.0`.
```

Comment:
Yep, I think this captures the heart of the problem, that we're kind of using CIDR format to express two things here. And also really like that you explicitly call out that you're getting `10.10.21.10` as your IP address :+1: